plugins/UM3NetworkPrinting/resources/qml/MonitorQueue.qml

```diff
@@ -186,17 +186,7 @@ Item
                 }
                 printJob: modelData
             }
-            model:
-            {
-                // When printing over the cloud we don't recieve print jobs until there is one, so
-                // unless there's at least one print job we'll be stuck with skeleton loading
-                // indefinitely.
-                if (Cura.MachineManager.activeMachineIsUsingCloudConnection || OutputDevice.receivedPrintJobs)
-                {
-                    return OutputDevice.queuedPrintJobs
-                }
-                return [null, null]
-            }
+            model: OutputDevice.queuedPrintJobs
```

Comment:
It breaks skeleton loading, but the loading times are so fast now that you don't notice it. We could bring it back, but I'd rather use a property on the output device that that indicates if a status has ever been received or not.

Reply:
But that property will only determine if we should show skeleton loading or not, right? It won't actually tell you if we have the data for x number of print jobs yet, since we don't know that until we actually have the print job data.

Reply:
Correct.

Reply:
But that's essentially how it worked anyway; when that property changed, we swap out the dummy model with null data for the actual model. In the not-yet-recieved state you'll still need to show skeletons somehow and I quite like the fact that we used the very same component for skeletons, but which showed grey rectangles for any missing information.

Reply:
I can put it back, but it seemed like a very implicit thing to me.